sentry/src/main/java/io/sentry/Baggage.java

```diff
@@ -15,7 +16,7 @@
 @ApiStatus.Experimental
 public final class Baggage {
 
-  static final @NotNull String CHARSET = "UTF-8";
+  static final @NotNull String CHARSET = StandardCharsets.UTF_8.toString();
```

Comment:
For some reason this is not being detected, I guess AGP regressed here.

> Warning: Lint will treat :sentry as an external dependency and not analyze it.
>  * Recommended Action: Apply the 'com.android.lint' plugin to java library project :sentry. to enable lint to analyze those sources.

The linter is able to analyze and check everything but does not consider the `minApi` apparently.

Reply:
I should revert this change and merge it anyway since I found out other issues with the linter.

Reply:
So this means we'll have to be careful and check manually in Android Studio if there's any minApi breakages we introduced?

Reply:
If we don't find a fix for this, yes, which sucks, this used to work.
@romtsn ideas?

Reply:
answered in https://github.com/getsentry/sentry-java/pull/2117/files#r902318872

Reply:
@romtsn where do you usually file your AGP bugs? I've created a repro https://github.com/marandaneto/NewApiLintTest

Reply:
This used to work in the past, so I'd consider either a bug or a non-documented breaking change.

Reply:
Yeah, that's strange, I'm pretty sure that it worked in the past when developing sentry-android v2 unless I'm mistaken, I don't recall leaking NewApi tho so I strongly think that it worked in the past.
We could try this workaround, it's a good idea.
I just quickly tried and it requires a bit of more work, I'd stop here tho, feel free to add this to our backlog and eventually prioritize finishing up.

Reply:
Sounds good, @adinauer or I could take it from there, thanks 👍